packages/teleterm/README.md

```diff
@@ -119,21 +119,21 @@ $ rm -rf ./../webapps/packages/teleterm/src/services/tshd/v1/ && cp -R lib/telet
   +---------------+   | tls/tcp on localhost     | |
   |    local      |   |                          | |
   | user profile  |   |                          v v
-  |   (files)     |   |                   +------+-+-------------------+
```

Comment:
What did you use to modify this diagram? I've used Monodraw in the past but idk if you can just import those diagrams like that there.

Reply:
FWIW, when I have to draw a non-sequential diagram, I typically use Graphviz and then put it through https://dreampuf.github.io/GraphvizOnline/

Monodraw is nice but I hated that I had to spend so much time moving boxes around when Graphviz can do that for me (though it's not always ideal and requires some additional work).